README.rst

```diff
@@ -40,6 +40,8 @@
 **scikit-learn** is a Python module for machine learning built on top of
 SciPy and is distributed under the 3-Clause BSD license.
 
+This is a test for the labeler.
+
```

Comment:
Can I review here?

Reply:
You're welcome! But this still doesn't work... :(
Thanks! :)